pkg/graphqljsonschema/jsonschema_test.go

```diff
@@ -10,6 +10,18 @@ import (
 	"github.com/wundergraph/graphql-go-tools/internal/pkg/unsafeparser"
 )
 
+func prettyPrint(s string) string {
+	var v interface{}
```

Comment:
it could be simplified to
```
func prettyPrint(s string) string {
	var out bytes.Buffer
	if err := json.Indent(&out, []byte(s), "", "  "); err  != nil {
		panic(err)
	}
	return out.String()
}
```

Reply:
Sorry, I missed this. I'll submit it as a separate PR.